qtUi/CMakeLists.txt

```diff
@@ -15,20 +15,11 @@ find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Widgets)
 find_package(Qt6 REQUIRED COMPONENTS WebEngineWidgets Core)
 find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Widgets)
 
-set(PROJECT_SOURCES
```

Comment:
Avoid defining file lists this way and instead prefer `target_sources`, this was the old way of doing things in CMake before version 2.8 when targets were introduced.  Since then `target_source` should always be preferred, background on why here: https://crascit.com/2016/01/31/enhanced-source-file-handling-with-target_sources/